src/renderer/Dataframe.js

```diff
@@ -134,14 +134,18 @@ export default class Dataframe extends DummyDataframe {
         this.height = height;
 
         this.propertyTex[propertyName] = gl.createTexture();
-        gl.bindTexture(gl.TEXTURE_2D, this.propertyTex[propertyName]);
-        gl.texImage2D(gl.TEXTURE_2D, 0, gl.ALPHA,
-            width, height, 0, gl.ALPHA, gl.FLOAT,
-            propertiesFloat32Array);
-        gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_S, gl.CLAMP_TO_EDGE);
-        gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_WRAP_T, gl.CLAMP_TO_EDGE);
-        gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MIN_FILTER, gl.NEAREST);
-        gl.texParameteri(gl.TEXTURE_2D, gl.TEXTURE_MAG_FILTER, gl.NEAREST);
+
+        if (propertiesFloat32Array) {
```

Comment:
Are we uploading the properties in another part?

Reply:
Not that I am aware of, buy I'll check other uses of `gl.texImage2D`